pallets/loans/src/mock.rs

```diff
@@ -172,11 +175,13 @@ impl pallet_pools::Config for MockRuntime {
 	type NAV = Loans;
 	type TrancheToken = TrancheToken<MockRuntime>;
 	type Time = Timestamp;
+	type ChallengeTime = ChallengeTime;
+	type MinUpdateDelay = MinUpdateDelay;
+	type RequireRedeemFulfillmentsBeforeUpdates = RequireRedeemFulfillmentsBeforeUpdates;
```

Comment:
I would prefer this to be a trait like 

```rust
trait UpdateGuard {
   released(pool: &PoolDetails, update: &ScheduledUpdateDetails) -> bool;
}
```

Then we could implement the release checking logic on a per runtime basis and also be more flexible with respect to changing the logic of when we allow/dissallow updates to be released.

Reply:
This is done now.